tasks/upload-sbom-to-atlas/README.md

```diff
@@ -6,11 +6,16 @@ Supports both CycloneDX and SPDX format.
+| sbomDir                   | Directory containing SBOM files. The task will search for CycloneDX JSON SBOMs recursively in this directory and upload them all to Atlas. The path is relative to the 'data' workspace.                                             | No       | None                                                                          |
 | httpRetries               | Max HTTP retry count.                                                                                                                                                                                                                | Yes      | 3                                                                             |
 | atlasSecretName           | Name of the Secret containing SSO auth credentials for Atlas.                                                                                                                                                                        | Yes      | atlas-prod-sso-secret                                                         |
 | bombasticApiUrl           | URL of the BOMbastic API host of Atlas.                                                                                                                                                                                              | Yes      | https://sbom.atlas.devshift.net                                               |
 | ssoTokenUrl               | URL of the SSO token issuer.                                                                                                                                                                                                         | Yes      | https://auth.redhat.com/auth/realms/EmployeeIDP/protocol/openid-connect/token |
-| skipUploadingSbom         | If enabled, tasks are skipped and no SBOM is uploaded. User must explicitly enable SBOM uploading by skipUploadingSbom: false.                                                                                                       | Yes      | true                                                                          |
 | supportedCycloneDxVersion | If the SBOM uses a higher CycloneDX version, `syft convert` in the task will convert all SBOMs to this CycloneDX version before uploading them to Atlas. If the SBOM is already in this version or lower, it will be uploaded as is. | Yes      | 1.4                                                                           |
 | supportedSpdxVersion      | If the SBOM uses a higher SPDX version, `syft convert` in the task will convert all SBOMs to this SPDX version before uploading them to Atlas. If the SBOM is already in this version or lower, it will be uploaded as is.           | Yes      | 2.3                                                                           |
+
+## Changes in 0.3.0
```

Comment:
There was never a 0.2.0. Why not just make this 0.2.0 with both changes in it and not falsely list what existed in 0.2.0 and call this 0.3.0?

Reply:
Just an oversight, I didn't realize this was still a single PR.